frontend/device/kobo/powerd.lua

```diff
@@ -335,8 +339,8 @@ function KoboPowerD:turnOnFrontlightHW()
         return
     end
     ffiUtil.runInSubProcess(function()
-        for i = 1,5 do
-            self:setIntensityHW(math.ceil(self.fl_min + ((self.fl_intensity * (1/5)) * i)))
+        for i = 1, 5 do
+            self:setIntensityHW(math.ceil(self.fl_min + (self.fl_intensity / 5 * i)))
```

Comment:
```suggestion
            self:setIntensityHW(math.ceil(self.fl_min + (self.fl_intensity / 5 * i))) -- @zwim don't change me :-P
```

Reply:
I dont know if it is important, but after applying these changes, Toggling frontlight on Kobo Libra 2 works as it should!
Thank You very much! :)